publications-2021.bib

```diff
@@ -353,6 +353,17 @@ @Article{2021:fraters.billen:on
   publisher = {American Geophysical Union ({AGU})}
 }
 
+@Article{2021:freese:numerical,
+  author  = {Freese, Jan Philip},
+  title   = {Numerical homogenization of time-dependent Maxwell's equations with dispersion effects},
+  year    = 2021,
+  doi     = {10.5445/IR/1000129214},
+  url     = {https://publikationen.bibliothek.kit.edu/1000129214},
+  language = {en},
+  publisher = {Karlsruher Institut f\"{u}r Technologie (KIT)},
+  copyright = {Open Access}
+}
```

Comment:
This should be marked up as `@PhDThesis`, not `@Article`. It needs to use `school=` instead of `publisher=`.

Reply:
True. I didn't pay attention. I will fix it.